### PR TITLE
videos round 1

### PR DESCRIPTION
**Pull request by guptstagram**
Have added some YouTube video links and few topics which I felt missing. Don't want to add random ones coming at the top of youtube search.

Will go through more videos with time for the listed topics and add.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,316 +1,588 @@
...
-  - [ ] `<head>` [Link](https://www.w3docs.com/learn-html/html-head-tag.html)
-  - [ ] `<body>` [Link](https://www.w3docs.com/learn-html/html-body-tag.html)
-  - [ ] `<script>` [Link](https://www.w3docs.com/learn-html/html-script-tag.html)
-  - [ ] `<style>` [Link](https://www.w3docs.com/learn-html/html-style-tag.html)
-  - [ ] `<link>` [Link](https://www.w3docs.com/learn-html/html-link-tag.html)
-  - [ ] `<meta>` [Link](https://www.w3docs.com/learn-html/html-meta-tag.html)
-  
+
+- [ ] HTML main tags [Playlist](https://www.youtube.com/playlist?list=PLblA84xge2_xNtaFnZhefjFbnDrpySKD3)
+
```
Comment (shrutikapoor08) [inline README.md:4]: Looks like there are formatting issues. Can you remove the extra spaces?

**Review by shrutikapoor08 — CHANGES_REQUESTED** [review]
Requesting formatting changes.

File: README.md (hunk 1 above)
Comment (guptstagram) [reply README.md:4]: Actually, prettier added those white spaces. Do you use some formating tool to format the markdown?

**Comment by guptstagram** [comment]
Made the changes.

**Review by shrutikapoor08 — APPROVED** [review]
Thanks a lot for your help

**Pull request merged**